include/libdnf/base/solver_problems.hpp

```diff
@@ -36,16 +36,17 @@ class SolverProblems {
     /// Provide information about package solver problems in a vector. Each problem can be transformed to string by
     /// package_solver_problem_to_string or all problems to a string by all_package_solver_problems_to_string().
     ///
-    /// @return Vector with structuralized package solver problems
+    /// @return Vector with structuralized package solver problems. The vector conteins solver errors represented
+    /// by a vector with std::pairs that describe each problem
     // @replaces libdnf/Goal.describeProblemRules(unsigned i, bool pkgs);
     // @replaces libdnf/Goal.describeAllProblemRules(bool pkgs);
-    std::vector<std::vector<std::pair<libdnf::ProblemRules, std::vector<std::string>>>> get_package_solver_problems();
+    std::vector<std::vector<std::pair<libdnf::ProblemRules, std::vector<std::string>>>> get_problems();
```

Comment:
Could you _please_ amend these in the original commits, instead of adding fixup commits on top that will not be squashed?

Reply:
Due to creation of p_impl the rename is not trivial. I would prefer to keep it in separate commit.